packages/react-composites/src/composites/common/AddPeopleDropdown.tsx

```diff
@@ -149,7 +149,7 @@ export const AddPeopleDropdown = (props: AddPeopleDropdownProps): JSX.Element =>
         )}
         {alternateCallerId && (
           <CallingDialpad
-            isMobile
+            isMobile={true}
```

Comment:
Off topic to this PR: UI components really shouldn't have an `isMobile` flag. They should have props that explicitly state what will happen.

Reply:
So this is needed so we know the touch event scenario and the long press handlers. maybe a rename to `isTouchDevice`?

Reply:
I only see it here:
![image](https://github.com/Azure/communication-ui-library/assets/2684369/704830b5-6178-4326-b140-83b350a62fb3)

If this is it I would name to just this specific purpose, something like `longPressTrigger: 'touch' | 'mouse' `